### ISSUE TITLE
Use capital 'A' in #include <arduino.h>

### ISSUE THREAD

**Issue by SukkoPera** [Rick-G1/FrequencyGenerator#1]
Hi and thanks for this nice library!

Just one thing: the [#include <arduino.h>](https://github.com/Rick-G1/FrequencyGenerator/blob/master/src/FrequencyGenerator.cpp#L112) at line 112 should be changed to #include <Arduino.h> with a capital 'A', since Linux and MacOS use case-sensitive filesystems and won't find the file otherwise.

**Comment by Rick-G1** [comment]
I'm glad someone got some benefit from my coding.  A lot of stuff like this I did while I was working was for circuit board test equipment manufacturer and was rather unique in its nature.   This was a modified byproduct of something I wrote for that. 

 

I have updated the file with the capital 'A', but not sure I got the update right as it's been over 2 years since I've done anything with github.. Hopefully the change is available now. 

 

Glad you liked it…. 

Rick Groome

 

From: Giorgioggì ***@***.*** 
Sent: Tuesday, September 12, 2023 12:13 PM
To: Rick-G1/FrequencyGenerator
Cc: Subscribed
Subject: [Rick-G1/FrequencyGenerator] Use capital 'A' in #include <arduino.h> (Issue #1)

 

Hi and thanks for this nice library!

Just one thing: the #include  <https://github.com/Rick-G1/FrequencyGenerator/blob/master/src/FrequencyGenerator.cpp#L112> <arduino.h> at line 112 should be changed to #include <Arduino.h> with a capital 'A', since Linux and MacOS use case-sensitive filesystems and won't find the file otherwise.
